src/plugins/kibana_react/public/saved_objects/saved_object_save_modal.tsx

```diff
@@ -230,10 +244,14 @@ export class SavedObjectSaveModal extends React.Component<Props, State> {
                 objectType: this.props.objectType,
                 confirmSaveLabel: (
                   <strong>
-                    <FormattedMessage
-                      id="kibana-react.savedObjects.saveModal.duplicateTitleDescription.confirmSaveText"
-                      defaultMessage="Confirm Save"
-                    />
+                    {this.props.confirmButtonLabel
+                      ? this.props.confirmButtonLabel
+                      : i18n.translate(
```

Comment:
Why is "confirm save" translated twice?

Reply:
Not sure tbh, I re-used this from the old implementation.